cluster/examples/kubernetes/ceph/operator.yaml

```diff
@@ -149,6 +149,10 @@ spec:
         - name: ROOK_DISABLE_DEVICE_HOTPLUG
           value: "false"
 
+        # Provide customised regex as the values using comma. If 'nil', default values will be used.
+        - name: DISCOVER_DAEMON_UDEV_BLACKLIST
+          value: "nil"
```

Comment:
Perhaps you can comment that with an example instead of `nil`?

Reply:
Okay. Thanks. I'll do that.